<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTShow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_13">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="1" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Compare Result</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBrowser" name="textBrowser_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
           <item row="1" column="2">
            <widget class="QTextBrowser" name="textBrowser_3"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>STREAM Rover Result</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>STREAM Base Result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>210</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>STREAM Rover</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget_3">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab_9">
                 <attribute name="title">
                  <string>Dynamic</string>
                 </attribute>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>IP</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_6">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>localhost</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>PORT</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_5">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>12346</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="text">
                     <string>Connect</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Stop</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>STREAM Base</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget_2">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab_7">
                 <attribute name="title">
                  <string>Dynamic</string>
                 </attribute>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>IP</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>localhost</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>PORT</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_2">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>12345</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>Connect</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Stop</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_8">
                 <attribute name="title">
                  <string>Static</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_12">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>X_CRD</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_z"/>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_y"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>Z_CRD</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_x"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Y_CRD</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QPushButton" name="pushButton_static">
                    <property name="text">
                     <string>Set</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Result Save</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Base Result</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_base">
                <property name="text">
                 <string>Base</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Rover Result</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_rover">
                <property name="text">
                 <string>Rover</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Compare Result</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_compare">
                <property name="text">
                 <string>Compare</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>SAVE</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Log</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>ENUPlot</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox_2">
              <property name="currentText">
               <string>1</string>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>0.1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QWidget" name="widget_4" native="true"/>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>FromZero</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>636</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>YAxis</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>SatNum</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>FromZero</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>636</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox_3">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QWidget" name="widget_6" native="true"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>YAxis</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>PDOP</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>FromZero</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QWidget" name="widget_7" native="true"/>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>636</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>YAxis</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Ratio</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="1">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>FYAxis</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>630</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>FromZero</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox_4">
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QWidget" name="widget_8" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_map" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsdfsdfsdf">
   <property name="text">
    <string>sdfsdfsdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
